certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/unitaccess/InMemoryCertificateUnitAccessEvaluationRepository.java

```diff
@@ -0,0 +1,35 @@
+package se.inera.intyg.certificateservice.infrastructure.unitaccess;
+
+import java.util.Collections;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Repository;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.CertificateType;
+import se.inera.intyg.certificateservice.domain.certificatemodel.repository.CertificateUnitAccessEvaluationRepository;
+import se.inera.intyg.certificateservice.domain.unitaccess.dto.CertificateAccessConfiguration;
+import se.inera.intyg.certificateservice.infrastructure.configuration.UnitAccessConfiguration;
+
+@Slf4j
+@Repository
+@RequiredArgsConstructor
+public class InMemoryCertificateUnitAccessEvaluationRepository implements
+    CertificateUnitAccessEvaluationRepository {
+
+  private final UnitAccessConfiguration unitAccessConfiguration;
+
+  @Override
+  public List<CertificateAccessConfiguration> get(CertificateType certificateType) {
```

Comment:
Maybe call it find instead. It will return any that is found but could be empty list.